spec/iasm.dd

```diff
@@ -26,6 +26,38 @@ $(SPEC_S D x86 Inline Assembler,
         $(D D_InlineAsm_X86_64) version identifiers, respectively.
     )
 
+$(H2 $(LNAME2 asmstatements, Asm statement))
+
+$(GRAMMAR
+$(GNAME AsmStatement):
+    $(D asm) $(GLINK FunctionAttributes)$(OPT) $(D {) $(GLINK AsmInstructionList)$(OPT) $(D })
+
+    $(GNAME AsmInstructionList):
+        $(GLINK AsmInstruction) $(D ;)
+        $(GLINK AsmInstruction) $(D ;) $(I AsmInstructionList)
+    )
+
+    $(P Assembler instructions must be located inside an `asm` block.
+        Like functions, `asm` statements must be anotated with adequate function attributes to be compatible with the caller.
```

Comment:
This sounds weird. I would just remove the entire sentence.

Reply:
No, this is the point of this documentation change. I meant that asm blocks must be annotated, just like local functions, despite of not being local functions.